src/app/GitCommands/Settings/FileSettingsCache.cs

```diff
@@ -54,6 +54,7 @@ protected FileSettingsCache(string settingsFilePath, bool autoSave = true)
             }
             catch
             {
+                _fileWatcher.EnableRaisingEvents = false;
```

Comment:
`_fileWatcher.EnableRaisingEvents` is `false` in this ctor

Reply:
The other occasions are similar, the value is set to false in the init even if not set in the ctor. So this should be handled the same.
Setting wsl first though